noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -711,59 +1221,14 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
-
-    <rule>
-        <title>Classification Systems count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT s.systemid series_system_id, COUNT(*) ks_count
-                    FROM arkivstruktur.arkivdel s
-                    JOIN arkivstruktur.klassifikasjonssystem ks ON ks._parent_id = s._id
-                    GROUP BY s.systemid
-                ]]>
-            </info>
-        </queries>
-    </rule>
-
-    <rule>
-        <title>Class count</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT ks.systemid ks_system_id, parent.systemid parent_system_id, COUNT(*) number_of_classes
-                    FROM arkivstruktur.klasse k
-                    LEFT JOIN arkivstruktur.klasse parent ON parent.systemid = k._ref_klasse
-                    LEFT JOIN arkivstruktur.klassifikasjonssystem ks ON ks.systemid = k._ref_klassifikasjonssystem
-                    GROUP BY ks.systemid, parent.systemid;
-                ]]>
-            </info>
-        </queries>
-    </rule>
-
-    <rule>
-        <title>Classes without subclasses, files, or records</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT ks._ref_arkivdel series_system_id, COUNT(DISTINCT(k.systemid)) classes_without_children
-                    FROM arkivstruktur.klasse k
-                    JOIN arkivstruktur.klassifikasjonssystem ks ON ks.systemid = k._ref_klassifikasjonssystem
-                    WHERE k.systemid NOT IN (SELECT _ref_klasse FROM arkivstruktur.klasse WHERE _ref_klasse IS NOT NULL GROUP BY _ref_klasse)
-                        AND k.systemid NOT IN (SELECT _ref_klasse FROM arkivstruktur.mappe WHERE _ref_klasse IS NOT NULL)
-                        AND k.systemid NOT IN (SELECT _ref_klasse FROM arkivstruktur.registrering WHERE _ref_klasse IS NOT NULL GROUP BY _ref_klasse)
-                    GROUP BY ks._ref_arkivdel;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST6">
         <title>File count</title>
+        <description>
+            Provides information about the number of Files per Series
```

Comment:
`Provides information about the number of files per series and tests whether the corresponding number specified in arkivuttrekk.xml is correct`